packages/@textlint/textlint-plugin-text/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // LICENSE : MIT
 "use strict";
 import { TextProcessor } from "./TextProcessor";
-module.exports = {
+export const TextPlugin = {
```

Comment:
The textlint plugin need to export `{ Processor }` object.

https://textlint.github.io/docs/plugin.html

Reply:
I found `textlint` test was failed for another reason.  It comes from wrong way of importing in "textlint-core.ts".

I fixed it at these commits.
32bcad1
d8f3432